samples/kotlin_compose/build.gradle.kts

```diff
@@ -49,18 +63,45 @@ android {
     }
 }
 
+val hiltVersion = "2.44.2"
+val composeVersion = "1.4.1"
+val coroutinesVersion = "1.5.2"
+val roomVersion = "2.4.2"
+
 dependencies {
 
     // compose compiler requires an updated version of kotlin
     implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.20"))
     implementation("androidx.core:core-ktx:1.10.0")
     implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.6.1")
     implementation("androidx.activity:activity-compose:1.7.1")
-    implementation(platform("androidx.compose:compose-bom:2022.10.00"))
     implementation("androidx.compose.ui:ui")
     implementation("androidx.compose.ui:ui-graphics")
     implementation("androidx.compose.ui:ui-tooling-preview")
     implementation("androidx.compose.material3:material3")
+
+    implementation(platform("androidx.compose:compose-bom:2022.10.00"))
+
+    // SDK
+    implementation(project(":sdk"))
+    implementation(project(":messagingpush"))
+    implementation(project(":messaginginapp"))
```

Comment:
This should be easier to test specific version of SDK. May be use `IS_DEVELOPMENT` flag for this? Open to other options as well, just want to keep it easier to use remote/local versions.